Update H69 for accurate 'accessibility supported'

mitchellevan opened the pull request:
~fixes~ does not fix, but moves in the right direction for #1712

mitchellevan committed:
Update H69 so the 'accessibility supported' assumption accurately matches normative WCAG

fixes #1712

mitchellevan committed:
The plug-in must itself be accessible

awkawk commented:
Bringing in cost here is problematic.

mitchellevan commented:
> Bringing in cost here is problematic.

My intention was to highlight normative WCAG, which does use the word "cost." I paraphrased by saying "no cost" instead of same cost, which in the content of browser and browser plug-ins I believe is accurate.

awkawk commented:
"The technology is supported in a widely-distributed plug-in that is also accessibility supported;"

There's no cost mentioned in this second option under the second condition of accessibility supported. The 4th option mentions cost but the 4th option isn't required. That's why it is problematic.

mitchellevan committed:
More clarity

Spell out the role of cost in the normative definition of 'accessibility supported'.

Assert that wide distribution is not and has not been the reality.

Clarify that these are examples of relevant scenarios, not part of a normative definition.

mitchellevan committed:
typo

patrickhlauke commented:
I'm actually wondering as well now if H69 should be made advisory, rather than sufficient, for Bypass Blocks - as otherwise we could say that of course it's sufficient to just mark up headings, since keyboard users (without other AT) that need to navigate by headings will have some plugin or specialised browser that lets them do that... but as that's not a common feature of browsers, and there's only a handful of obscure extensions, it seems a stretch to just allow headings and put onus on users to hunt those tools down
